modules/widgets/social-media-icons.php

```diff
@@ -109,7 +109,11 @@ public function widget( $args, $instance ) {
 		}
 
 		if ( ! empty( $instance['youtube_username'] ) ) {
-			$html .= '<li><a title="' . sprintf( $alt_text, esc_attr( $instance['youtube_username'] ), 'YouTube' ) . '" href="' . esc_url( 'https://www.youtube.com/channel/' . $instance['youtube_username'] ) . '" class="genericon genericon-youtube" target="_blank"><span class="screen-reader-text">' . sprintf( $alt_text, esc_html( $instance['youtube_username'] ), 'YouTube' ) . '</span></a></li>';
+			if ( substr( $instance['youtube_username'], 0, 2 ) == 'UC' ) {
```

Comment:
@dereksmart Are these currently supported, or are you just suggesting we should also try to take those into account while making this update?